scripts/build

```diff
@@ -105,7 +107,12 @@ STEP_DIR=$OBJ_GCC_STATIC
 if [[ ! -d $STEP_DIR ]]; then
   printf "${WHT}Compiling GCC static${CLR}${NWL}"
 
-  rmStep $OBJ_MUSL
+  rm -rf $OBJ_MUSL
+
+  # Fix for arm builds on osx
+  if [[ $(uname -s) = "Darwin" && $ARCH = "arm" ]]; then
+    export CXXFLAGS="$CXXFLAGS -fbracket-depth=512"
+  fi
```

Comment:
Ok, I was not sure if it was related to it, don't know why when I saw Darwind and ARM on the same line I first though about the iPhone :-P Yes, BigRedButton is generating cross-toolchains for x86, x64 y armv7 :-) NodeOS is still having problems with the ARM support, but there's there some work progress anyway :-)